src/Discord.Net.Core/CDN.cs

```diff
@@ -102,11 +102,12 @@ public static string GetDefaultUserAvatarUrl(ulong userId)
         /// </summary>
         /// <param name="guildId">The guild snowflake identifier.</param>
         /// <param name="iconId">The icon identifier.</param>
+        /// <param name="size">The size of the image to return in horizontal pixels. This can be any power of two between 16 and 2048.</param>
```

Comment:
> This can be any power of two between 16 and 2048.

Should we add a check for this instead of blindly adding it as the size param?

Reply:
It's the same behavior as in all other endpoints that have a `size` parameter